### PR TITLE
Add maven package export script

### PR DESCRIPTION
**Pull request by gokhanbarisaker**
`maven.gradle` allows to export archived package that is compatible with jcenter.

### CODE REVIEW

**Comment by gokhanbarisaker** [comment]
We may remove, since this script exports pom file identical to the one generated with `createPom` task.

File: lib/build.gradle [hunk 1]
```diff
@@ -58,4 +59,4 @@ task testRestSuite(type: Test) {
     logger.lifecycle("Running test: " + descriptor)
   }
   outputs.upToDateWhen { false }
-}
+}
```
Comment (paddybyers) [inline lib/build.gradle]: Newline pls

**Comment by mattheworiordan** [comment]
> We may remove createPom task, since this script exports identical pom file.

Ok, @gokhanbarisaker should we just do that as part of this PR then?

**Comment by mattheworiordan** [comment]
LGTM :+1:

**Comment by gokhanbarisaker** [comment]
> should we just do that as part of this PR then?

Sure, if possible I would like to delay this cleanup until I resolve #69.

**Comment by paddybyers** [comment]
LGTM. @gokhanbarisaker you ok for me to merge?

**Pull request merged**